writer/index.html

```diff
@@ -108,10 +108,23 @@
 
             return uniqueVariables.map(va => `<th>${va}</th>`).join('');
         }
+        function encodeHTMLEntities(code) {
+          var htmlEntities = {
+            "<": "&lt;",
+            ">": "&gt;",
+            "&": "&amp;",
+            "'": "&apos;",
+            '"': "&quot;",
+            };
+
+            return code.replace(/[<>&'"]/g, function (match) {
```

Comment:
Interesting how the first argument is handled.. Instead of string it is handing as regular expression?